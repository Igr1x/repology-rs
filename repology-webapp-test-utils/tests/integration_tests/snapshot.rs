// SPDX-FileCopyrightText: Copyright 2025 Dmitry Marakasov <amdmi3@amdmi3.ru>
// SPDX-License-Identifier: GPL-3.0-or-later

use super::*;

#[tokio::test]
async fn test_as_snapshot() {
    let resp = perform_mock_request("abc").await;
    insta::assert_snapshot!(resp.as_snapshot().unwrap());
}
