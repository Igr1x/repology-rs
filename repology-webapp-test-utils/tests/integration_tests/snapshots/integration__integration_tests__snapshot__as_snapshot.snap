---
source: repology-webapp-test-utils/tests/integration_tests/snapshot.rs
expression: resp.as_snapshot().unwrap()
snapshot_kind: text
---
Status: 200
Header: content-type: text/plain
Header: content-length: 3
---
abc
